### ISSUE TITLE
PHP Notice:  Trying to get property of non-object in redelivre/src/wp-content/themes/redeninja/page-capa.php

### ISSUE THREAD

**Issue by jacsonp** [ForaDoEixo/redeninja#19]
[Tue Apr 11 02:03:24.034351 2017] [:error] [pid 6620] [client xx] PHP Notice:  Trying to get property of non-object in redelivre/src/wp-content/themes/redeninja/page-capa.php on line 89, referer: http://compartilhamento.beta.redelivre.org.br/
[Tue Apr 11 02:03:24.034370 2017] [:error] [pid 6620] [client xx] PHP Stack trace:, referer: http://compartilhamento.beta.redelivre.org.br/
[Tue Apr 11 02:03:24.034377 2017] [:error] [pid 6620] [client xx] PHP   1. {main}() redelivre/src/index.php:0, referer: http://compartilhamento.beta.redelivre.org.br/
[Tue Apr 11 02:03:24.034390 2017] [:error] [pid 6620] [client xx] PHP   2. require() redelivre/src/index.php:17, referer: http://compartilhamento.beta.redelivre.org.br/
[Tue Apr 11 02:03:24.034394 2017] [:error] [pid 6620] [client xx] PHP   3. require_once() redelivre/src/wp-blog-header.php:19, referer: http://compartilhamento.beta.redelivre.org.br/
[Tue Apr 11 02:03:24.034398 2017] [:error] [pid 6620] [client xx] PHP   4. include() redelivre/src/wp-includes/template-loader.php:74, referer: http://compartilhamento.beta.redelivre.org.br/

**Comment by iwasthesword** [comment]
ForaDoEixo/redeninja/issues/19
ForaDoEixo/redeninja/issues/20
ForaDoEixo/redeninja/issues/21
São o mesmo bug em partes diferentes do codigo

```
$args = array(
	'numberposts' => 5,
	'category' => get_category_by_slug('destaque1')->term_id,
	'post_type' => 'post'
	);
```
Então se não houver a categoria ele não consegue acessar o `term_id`. Vou colocar uma checagem pra isso.

**Comment by iwasthesword** [comment]
#19 #20 #21

**Comment by iwasthesword** [comment]
Fix: 719f0e7
